jerry-core/debugger/debugger.c

```diff
@@ -26,6 +28,15 @@
 #ifdef JERRY_DEBUGGER
 
 /**
+ * HAVE_TIME
+ */
```

Comment:
this doc comment is not needed (or not needed in its current form). A `/**` is a doxygen comment for documenting an entity in the code. but we usually don't document the inclusion of headers. (even if we would, we should add descriptive English text in the documentation.)

TL;DR: drop this comment